packages/shorebird_cli/lib/src/commands/patch/patch_ios_command.dart

```diff
@@ -278,8 +278,11 @@ ${summary.join('\n')}
     return ExitCode.success.code;
   }
 
-  String get _aotOutputPath =>
-      p.join(Directory.current.path, 'build', 'out.aot');
+  String get _aotOutputPath => p.join(
+        shorebirdEnv.getShorebirdProjectRoot()!.path,
```

Comment:
Makes me wonder if there ends up being a ShorebirdEnv which is some sort of at-launch environment and later some sort of ProjectEnv which is resolved and we know has a project directory. 🤷